packages/compiler-core/__tests__/transforms/transformElement.spec.ts

```diff
@@ -836,6 +836,24 @@ describe('compiler: element transform', () => {
       })
     })
 
+    test('capitalized version w/ static binding', () => {
```

Comment:
I think we can just write the whole word 😆 
```suggestion
    test('capitalized version with static binding', () => {
```